docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
I'm just wondering what is best practise for spam-headers. Imagine you get external e-mails with `X-Spam-Flag: YES`. Now you pass mails directly via pipe to procmail or dovecot who do the filtering for you. To avoid confusion with tags our system adds, you must either remove or rename such tags. On the other hand, if the email was already considered as spam by the delivering mail server why accepting it in the first place?